### PR TITLE
Added 3rd survey state of finished

### PR DESCRIPTION
**Pull request by DavidSciola**
Changed survey states from
open -> closed
to
being edited -> being filled -> finished

### CODE REVIEW

**Comment by DavidSciola** [comment]
@DavidHaighton Note that I needed to change the application properties in main/src so that the app wouldn't use the actual Postgres db. I needed to do this because I changed the open/close bool of survey to an enum and the Postgres db is still using the open/close bool. At some point you will need to update the Postgres db.

**Review by Shashaank5 — APPROVED** [review]
Looks good overall. What was the purpose of adding the new columns finish and status instead of availability

**Review by DavidHaighton — CHANGES_REQUESTED** [review]
small  changes

File: src/main/java/com/based/basedsurvey/controller/FillSurveyController.java [hunk 1]
```diff
@@ -35,9 +35,15 @@ public String getAnswer(@PathVariable Long surveyID, Model model) {
         Survey s = this.surveyRepository.findSurveyById(surveyID);
         model.addAttribute("surveyName", s.getName());
 
-        // if survey is closed
-        if (!s.isOpen()) {
-            model.addAttribute("issue", "Survey is closed");
+        // if survey is still being edited
```
Comment (DavidHaighton) [inline src/main/java/com/based/basedsurvey/controller/FillSurveyController.java:38]: need updated uml and readme

File: src/test/resources/application.properties [hunk 2]
```diff
@@ -1,4 +1,4 @@
-spring.datasource.driver-class-name=org.h2.Driver
-spring.datasource.url=jdbc:h2:mem:db;DB_CLOSE_DELAY=-1
-spring.datasource.username=sa
-spring.datasource.password=sa
\ No newline at end of file
+#spring.datasource.driver-class-name=org.h2.Driver
+#spring.datasource.url=jdbc:h2:mem:db;DB_CLOSE_DELAY=-1
+#spring.datasource.username=sa
```
Comment (DavidHaighton) [inline src/test/resources/application.properties:3]: ill fix the postgress stuff first and then approve later

**Comment by DavidSciola** [comment]
> Looks good overall. What was the purpose of adding the new columns finish and status instead of availability

I basically just renamed the availability column to status and the new finish column contains the button to finish a survey.

File: src/main/java/com/based/basedsurvey/controller/FillSurveyController.java (hunk 1 above)
Comment (DavidSciola) [reply src/main/java/com/based/basedsurvey/controller/FillSurveyController.java:38]: done

**Comment by DavidHaighton** [comment]
i cleared the entire production database lmao

**Review by DavidHaighton — APPROVED** [review]

**Pull request merged**